package.json

```diff
@@ -15,7 +15,8 @@
     "check": "beachball check --changehint \"Run 'npm run change' to generate a change file\"",
     "invoke-publish": "cross-env-shell beachball publish --yes --access public --message \\\"applying package updates [skip ci]\\\" -n $NPM_SECRET_TOKEN",
     "validate": "npm run build && npm run lint && npm run test",
-    "performance": "lhci autorun"
+    "performance": "lhci autorun",
+    "wafer-performance": "lhci autorun --collect.staticDistDir='./packages/nimble-components' --collect.numberOfRuns=1 --collect.url='http://localhost:58452/performance/index.html'"
```

Comment:
Can you configure these parameters in a `.lighthouserc.js` file like [we do here](https://github.com/ni/nimble/blob/main/lighthouserc.js)? One way to do this would be to put a new lighthouserc file in the `nimble-components/performance` directory and then change this command to set that as the working directory before running `lhci`.

Rationale: keeping configuration consistent makes it easier to understand and we generally prefer config files over command line flags since it's easier to read

Reply:
did it